libs/deserializer/src/main/java/de/cyface/deserializer/BinaryFormatDeserializer.java

```diff
@@ -101,13 +101,13 @@ public Measurement read() throws IOException, InvalidLifecycleEvents, Unsupporte
                     .directions(measurement.getDirectionsBinary().getDirectionsList());
             final var builder = new TrackBuilder();
             final var tracks = builder.build(locations, events, accelerations, rotations, directions,
-                    metaData.getIdentifier());
+                    metaData.identifier);
             return new Measurement(metaData, tracks);
         }
     }
 
     @Override
     public List<MeasurementIdentifier> peakIntoDatabase() {
-        return Collections.singletonList(metaData.getIdentifier());
+        return Collections.singletonList(metaData.identifier);
```

Comment:
Isn't this Java Code? In that case `getIdentifier()` was correct. Avoiding this is one of the reasons switching to Kotlin. ;)
Anyways, `BinaryFormatDeserializer` should not directly access a metaData attribute. Why Isn't this private?

Reply:
Removed `@JvmField` from `MetaData.kt` properties to ensure Java uses the generated getter/setter methods instead of accessing the fields directly.